app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
```

Comment:
do we have configuration for this somewhere else? And does Hyrax make active use of ActionMailer for notifications? Essentially, I'm wondering about having a from address hard-coded here